src/main/java/com/tradeshift/service/HelloWorldInterface.java

```diff
@@ -0,0 +1,6 @@
+package com.tradeshift.service;
+
+
+public interface HelloWorldInterface {
+    public String getHelloMsg(String name);
```

Comment:
No need to abbreviate names. That's what ctrl+space in the IDE is for. `getHelloMessage` or just `getMessage` is better.